gandi/cli/commands/domain.py

```diff
@@ -38,6 +38,18 @@ def info(gandi, resource):
 
 
 @cli.command()
+@click.argument('resource', metavar='DOMAIN', required=False)
+@pass_gandi
+def check(gandi, resource):
+    """Check domain availability and price."""
+
+    result = gandi.domain.check(resource)
```

Comment:
You should use an explicite variable name here, `domain` instead of `result`
And you don't need to provide the justify argument as it is the default value.